lib/suse/toolkit/utilities.rb

```diff
@@ -21,7 +21,8 @@ def system_auth
 
       rescue
         raise SUSE::Connect::CannotBuildBasicAuth,
-              "Cannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}"
+              "\nCannot read username and password from #{SUSE::Connect::Credentials::GLOBAL_CREDENTIALS_FILE}. " \
+              "Has this system been registered already?"
```

Comment:
I believe it only gets raised if there is no credentials file to be found, in my case it looks for `/etc/zypp/credentials.d/SCCcredentials`. From what I saw, this is the case when the system has not been registered, but I can be mistaken of course. I will see if I find a better solution.